### DISCUSSION TITLE
Can't build Uno.UI-Skia-only.slnf

### DISCUSSION THREAD

**Discussion by zxslzx** [unoplatform/uno#7819] [Q&A]
I build Uno.UI-Skia-only.slnf project in vs2022, Compilation error occurred
![image](https://user-images.githubusercontent.com/32009362/149435059-eca886fd-0065-4970-aa27-f1834355277e.png)
![image](https://user-images.githubusercontent.com/32009362/149435123-fd9c818e-0804-4456-a255-e0de6f67d141.png)

**Comment by workgroupengineering** [comment]
Have you read [this ](https://platform.uno/docs/articles/uno-development/building-uno-ui.html) guide?

**Reply by zxslzx** [reply]
yes

**Reply by jeromelaban** [reply]
This looks like a nuget restore issue. Can you try building the slnf using the command line? `msbuild /r /bl Uno.UI-Skia-only.slnf` and see if it works better?

You can troubleshoot the result using https://platform.uno/docs/articles/uno-builds-troubleshooting.html?q=binlog#generating-msbuild-binary-log-files

**Reply by zxslzx** [reply]
Msbuild / r / bl Uno.UI-Skia-only.slnf   it can work properly.
